### DISCUSSION TITLE
The username and password are visible in the configuration file sql_exporter.yml in the data_source_name: string. Is there a way we can hide these credentials? I could see a solution for AWS EC2 but I am using windows VM in Azure.

### DISCUSSION THREAD

**Discussion by Nadzee** [burningalchemist/sql_exporter#311] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by burningalchemist** [comment, marked as answer]
Hey @Nadzee, please have a look at this issue, you might find suitable options: https://github.com/burningalchemist/sql_exporter/issues/251

As for AWS, Secret Manager integration was added by an engineer from AWS. I'd like to introduce more generic way to inject secrets in the future, but my capacity is limited.
